dask_cuda/cli.py

```diff
@@ -324,6 +337,12 @@ def main(
     multiprocessing_method,
     **kwargs,
 ):
+    """Launch a distributed worker with GPUs attached to an existing SCHEDULER.
```

Comment:
Not a big deal, but are there any reasons for `SCHEDULER` to be all uppercase? Is it referring to something that is normally written in uppercase? If not, I would suggest rewriting this.

Reply:
```suggestion
    """Launch a distributed worker with GPUs attached to an existing dask scheduler.
```

Reply:
My assumption was that it was to make it clear that it corresponded to the `SCHEDULER` argument that crops up in the `--help` output:

```
Usage: dask cuda worker [OPTIONS] [SCHEDULER] [PRELOAD_ARGV]...

  Launch a distributed worker with GPUs attached to an existing SCHEDULER.
```

It looks like this is also done for `dask worker`, though I can see how this description could seem somewhat strange from the `dask cuda` output.

Is there a way in click to render documentation for arguments? If so, that could be a good place to give more detail on what the expected input for `SCHEDULER` is; if not, I guess a reasonable compromise is to take your suggestion and dump info about the `SCHEDULER` argument in the subsequent lines of the docstring so that it only appears when running `dask cuda worker`.

Reply:
I see your point, it feels a little unusual but I'm ok with either one, if your prefer we can leave it as it is. I have no idea about click rendering documentation, not exactly sure how that would even work on the CLI, so my guess is that this probably doesn't exist.